src/org/sosy_lab/java_smt/api/FloatingPointFormulaManager.java

```diff
@@ -345,36 +353,118 @@ FloatingPointFormula multiply(
 
   /**
    * Create a term for assigning one floating-point term to another. This means both terms are
-   * considered equal afterwards. This method is the same as the method <code>equal</code> for other
-   * theories.
+   * considered equal afterward, based on their bit pattern (i.e., <code>0.0 != -0
+   * .0</code> and <code>NaN ==/!= NaN</code>, depending on the bit pattern of each NaN). This
+   * method is the same as the method <code>equal</code> for other theories.
    */
   BooleanFormula assignment(FloatingPointFormula number1, FloatingPointFormula number2);
 
   /**
    * Create a term for comparing the equality of two floating-point terms, according to standard
-   * floating-point semantics (i.e., NaN != NaN). Be careful to not use this method when you really
-   * need {@link #assignment(FloatingPointFormula, FloatingPointFormula)}.
+   * floating-point semantics (i.e., <code>NaN != NaN</code> and <code>0.0 == -0.0</code>). Be
+   * careful to not use this method when you really need {@link #assignment(FloatingPointFormula,
```

Comment:
Maybe an example that shows the difference of this method and `assignment()` would be nice.

Reply:
The example is in the documentation:
- assignment: `NaN ==/!= NaN` and `0.0 != -0.0`
- FP-equality: `NaN != NaN` and `0.0 == -0.0`

Reply:
Perfect!